addons/mail/static/src/core/common/quick_reaction_menu.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
     <t t-name="mail.QuickReactionMenu">
-        <Dropdown state="dropdown" manual="true" menuClass="`o-mail-QuickReactionMenu-popover shadow-none`" position="'top-middle'" navigationOptions="navigationOptions">
-            <button class="btn border-0 p-0 o-inline" t-att-class="attClass" t-att-title="props.action.name" t-att-aria-label="props.action.name" t-on-click="onClick">
+        <Dropdown state="dropdown" manual="true" menuClass="`o-mail-QuickReactionMenu-popover shadow-none`" position="'top-middle'" navigationOptions="navigationOptions" bottomSheet="false">
```

Comment:
Small change from previous preview: don't enable bottomSheet in mobile as the quick menu looks best without it:
<img width="377" height="671" alt="Screenshot 2025-09-18 at 20 38 06" src="https://github.com/user-attachments/assets/f7460631-95d8-4145-bc62-d71cc549b32c" />
<img width="378" height="667" alt="Screenshot 2025-09-18 at 20 38 24" src="https://github.com/user-attachments/assets/fd6f59f8-14e3-4c1b-8efd-4e784abe8565" />